en/ros/ros2_offboard_control.md

```diff
@@ -76,7 +76,7 @@ Besides that:
 
 ## Implementation
 
-The source code of the offboard control example can be found in [offboard_control.cpp](https://github.com/PX4/px4_ros_com/blob/master/src/examples/offboard/offboard_control.cpp).
+The source code of the offboard control example can be found in [offboard_control.cpp](https://github.com/PX4/px4_ros_com/blob/main/src/examples/offboard/offboard_control.cpp).
```

Comment:
FYI, also this page has to be heavily modified as a consequence of the new bridge.

Reply:
Yes. 

As above, we can do that. Lets work though the bits, starting with installation instructions.